examples/gnrc_tftp/Makefile

```diff
@@ -0,0 +1,42 @@
+# name of your application
+APPLICATION = gnrc_tftp
+
+# If no BOARD is found in the environment, use this default:
+BOARD ?= native
+
+# This has to be the absolute path to the RIOT base directory:
+RIOTBASE ?= $(CURDIR)/../..
+
+BOARD_INSUFFICIENT_MEMORY := airfy-beacon chronos msb-430 msb-430h nrf51dongle \
```

Comment:
msp430 builds keep failing due to not being able to parse `typedef enum { ... } enum_t` what do I do with those?